src/components/CookieConsentBanner.js

```diff
@@ -0,0 +1,161 @@
+    }
+  }, [hasTrackedView, acceptanceCookie])
+
+  const updateAcceptanceCookie = () => {
+    setIsBannerOpen(false)
+    setAcceptanceCookie(Cookies.get(ACCEPTANCE_COOKIE))
+  }
+
+  const acceptCookies = () => {
+    Cookies.set(ACCEPTANCE_COOKIE, true, { expires: 28 * 3 })
+    updateAcceptanceCookie()
+  }
+
+  const declineCookies = () => {
+    Cookies.set(ACCEPTANCE_COOKIE, false)
+    updateAcceptanceCookie()
+  }
+
+  const acceptanceCookieExists = typeof acceptanceCookie !== 'undefined'
+
+  const displayBanner = !acceptanceCookieExists || isBannerOpen
+
+  return (
+    <div>
+      <div aria-expanded={displayBanner}>
+        {displayBanner && (
+          <div className="cookie-banner">
+            <h1 id="cookie-content-p1">
+              Can we use cookies to help us improve this site?
+            </h1>
+            <div className="cookie-content-p2">
+              We'd like to use Google Analytics cookies to collect and report
+              information on how people use the site. We will use this to help
+              us improve our website.
+            </div>
+            <div className="cookie-content-p2">
+              Allowing us to use cookies does not allow us to identify you. For
+              more information please see our{' '}
+              <a href="https://www.neontribe.co.uk/privacy-policy">
+                'Privacy Policy'
+              </a>{' '}
+              page.
+            </div>
+            <div>
+              <Button onClick={acceptCookies}>Accept</Button>
+              <Button onClick={declineCookies}>Decline</Button>
+            </div>
+          </div>
+        )}
+      </div>
+
+      {!displayBanner && (
+        <button
+          onClick={() => setIsBannerOpen(true)}
+          className="cookie-sidebar-button"
+          aria-controls="cookie-banner"
+        >
+          <p>Cookie</p>
+        </button>
+      )}
+
+      <style jsx>{`
+        .cookie-banner {
+          //nav
+          display: flex;
+          flex-direction: column;
+          /*justify-content: center;*/
+          background: #effffa;
+          height: 100%;
+          position: fixed;
+          top: 0;
+          //words editting
+          padding: 2rem;
+          letter-spacing: 0;
+          color: black;
+          text-decoration: none;
+          width: 400px;
+        }
+
+        @media (max-width: 576px) {
+          width: 100%;
+        }
+
+        .cookie-sidebar-button {
+          height: 100%;
+          width: 1.5rem;
+          position: fixed;
+          top: 0;
+          background-color: transparent;
+          transition: 0.5s;
+        }
+
+        p {
+          transform: rotate(90deg);
+          color: white;
+        }
+
+        h1 {
+          font-size: 1.5rem;
+        }
+
+        .cookie-content-p2 {
+          letter-spacing: 0.015em;
+          font-weight: normal;
+          disaply: inline-block;
+          line-height: 1.5;
+          padding: 5px 0px;
```

Comment:
We can wrap the two paragraphs in the `<Text>` element from Text.js file so that the styling is the same as in other places.